lms/static/js/fixtures/courseware/link_clicked_events.html

```diff
@@ -0,0 +1,3 @@
+<a class="external-link" href="http://some.website.com/" />
```

Comment:
http://website.com actually exists.  To avoid sending them unwanted traffic, use http://example.com instead, as that domain specifically exists for the purpose of being used as a generic target.